Userland/Libraries/LibDisassembly/riscv64/Formatting.h

```diff
@@ -426,6 +427,26 @@ struct AK::Formatter<Disassembly::RISCV64::Branch::Condition> : AK::Formatter<St
     }
 };
 
+template<>
+struct AK::Formatter<Disassembly::RISCV64::CSRInstruction::Operation> : AK::Formatter<StringView> {
+    ErrorOr<void> format(FormatBuilder& builder, Disassembly::RISCV64::CSRInstruction::Operation op)
+    {
+        auto op_name = ""sv;
+        switch (op) {
```

Comment:
Maybe sprinkle some `using`/`using enum`s in the formatters
The namespace lookup chains assume alarming proportions